src/Components/PostsDisplay.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useNavigate } from "react-router-dom";
+
+function PostsDisplay({
+  messages,
+  startUpdate,
+  deleteMessage,
+  addLike,
+  addDislikes,
+  isLoggedIn,
+  user
+}) {
+  const navigate = useNavigate()
+  const tdStyle = { border: "1px solid white", padding: "10px" };
+
+  let messageListItems = (
+    <table style={{ borderCollapse: "collapse", width: "100%" }}>
+      <thead>
+        <tr>
+          <th style={tdStyle}>Name</th>
+          <th style={tdStyle}>Post</th>
+          <th style={tdStyle}>Timestamp</th>
+          <th style={tdStyle}>Edit</th>
+          <th style={tdStyle}>Delete</th>
+          <th style={tdStyle}>Picture</th>
+          <th style={tdStyle}>Like</th>
+          <th style={tdStyle}>Dislike</th>
+        </tr>
+      </thead>
+      <tbody>
+        {messages.map((message) => (
+          <tr key={message.key}>
+            <td style={tdStyle}>{message.val.name}</td>
+            <td style={tdStyle}>{message.val.text}</td>
+            <td style={tdStyle}>
+              <div>{message.val.timeStamp}</div>
+            </td>
+            <td style={tdStyle}>
+              {isLoggedIn && message.val.name == user.email ? (
+                <button onClick={() => {
+                  startUpdate(message)
+                  navigate("/form")
+                } }>Edit</button>
+              ) : null}
+            </td>
+            <td style={tdStyle}>
+              {isLoggedIn && message.val.name == user.email ? (
+                <button onClick={() => deleteMessage(message.key)}>
+                  Delete
+                </button>
+              ) : null}
+            </td>
+            <td style={tdStyle}>
```

Comment:
images are generally self closing tags
<img src={message.val.url} alt=""
                width="200px"
                height={"200px"} />
                
Also I apply style on images using the style sheet and a class.
Or you could use the style prop as opposed to width and height.